timeline/transform.py

```diff
@@ -59,15 +60,31 @@ def add_sexp_strs(timeline):
     Returns:
         list: The updated JSON data.
     """
+    def strip_comments(line):
```

Comment:
the old approach didn't properly handle semi-colons inside of strings.
(Example line: `(func (Func "swap") (Expr "r") (Type "void") (Type "{int *x; int *y;}*"))`)